.gitignore

```diff
@@ -42,13 +42,6 @@ lib/measures/*/tests/output
 /lib/measures/test_results
 /lib/measures/.rubocop*
 /lib/measures/*/tests/tbd.json
-json/*out.json
-spec/files/osms/tbd.out.json
-os_model_KIVA.osm
-warehouse_KIVA.osm
-os_model_test.osm
-os_model_FD.osm
-*.md
 
 runner.conf
 
```

Comment:
@macumber: Shuffled things around in the .gitignore file, created folders for generated files (to Git ignore), etc.